lua/nvim-treesitter/parsers.lua

```diff
@@ -411,6 +411,17 @@ list.c_sharp = {
   maintainers = { "@Luxed" },
 }
 
+list.todotxt = {
+  install_info = {
+    url = "https://github.com/arnarg/tree-sitter-todotxt.git",
+    files = { "src/parser.c" },
+    branch = "main",
+  },
+  filetype = "todotxt",
```

Comment:
is this a legit filetype ? @theHamsta i think we don't accept these kind of parsers right ? Not sure about that. Otherwise it would be good to go.

Reply:
We have no filetype detects but that would be fine for since it doesn't interfere with anything. We could also omit the filetype since the language of the parser is `todotxt` already.